### ISSUE TITLE
H96: Using the track element to provide audio descriptions

### ISSUE THREAD

**Issue by josephlabrecque** [dulibrarytech/digitaldu-frontend#157]

**Comment by josephlabrecque** [comment]
Text descriptions can be included in audio and video within Kaltura - but audio files cannot be added as tracks to video files.
